webui_runner/plugs/wamr_app_aot/echo_branch.sh

```diff
@@ -0,0 +1 @@
+echo main
```

Comment:
To avoid the hard-code, could this be read out of the downloaded Git directory using `git branch --show-current`?

Reply:
I'm guessing this was a quick hack that was never implemented.

Reply:
Logically a different patch. Can address in another patch that targets all VMs?

Reply:
https://github.com/bytecodealliance/sightglass/issues/32